go.mod

```diff
@@ -19,6 +19,7 @@ require (
 )
 
 require (
+	github.com/Azure/AgentBaker/pkg/proto/gen/nbcontractv1 v0.0.0-20231024012513-889e5d49d683 // indirect
```

Comment:
a bit confused why we're now requiring this

Reply:
You are right. This is not needed. Ran "go mod tidy" to clean it